README.rst

```diff
@@ -350,6 +350,62 @@ shows the different configuration options available:
         # image is passed to subsequent steps.
         import: path/to/image/archive.tar
 
+        # Specify the secrets that should be used when building your image,
+        # similar to the --secret option used by Docker
+        # More info about secrets: https://docs.docker.com/build/building/secrets/
+        secrets:
+          # Example of a secret that is a file
```

Comment:
we were going to originaly take this string version or a map version breaking the different pieces of this string out. I'm okay with just this string but wanted to double check if thats what we are doing now.

Reply:
Was about to ask the same thing but you beat me to it :)

Reply:
Yeah I opted for the more simple and straightforward solution of a list of strings which match the syntax used by docker. In the ticket it sounds like we decided we could either do a list of strings OR a dictionary. I am not sure the list dictionary adds too many benefits and adds more areas were bugs can occur.

Reply:
I agree and am for this syntax since it matches Docker anyways.